modules/table/src/main/java/org/apache/ignite/internal/table/distributed/disaster/DisasterRecoveryManager.java

```diff
@@ -278,10 +277,16 @@ public DisasterRecoveryManager(
         nodeLeftListener = new LogicalTopologyEventListener() {
             @Override
             public void onNodeLeft(LogicalNode leftNode, LogicalTopologySnapshot newTopology) {
-                MultiNodeOperations operations = operationsByNodeName.get(leftNode.name());
-                if (operations != null) {
-                    operations.completeAllExceptionally(leftNode.name(), new NodeStoppingException());
-                }
+                operationsByNodeId.compute(leftNode.id(), (node, operations) -> {
+                    if (operations != null) {
+                        operations.completeAllExceptionally(
+                                leftNode.name(),
+                                new NodeNotFoundException(Set.of(leftNode.name()))
```

Comment:
This exception class comes from the compute module, so it seems weird here. How about adding an exception class specific to disaster recovery, or a common exception with the same meaning? But it should be not about 'node not found', but rather 'node left'. Or maybe we already have such an exception class?